modules/fedimint-wallet-client/src/lib.rs

```diff
@@ -291,11 +470,64 @@ impl WalletClientModule {
 
         (operation_id, deposit_sm, address)
     }
+
+    pub async fn get_withdraw_fees(
+        &self,
+        address: bitcoin::Address,
+        amount: bitcoin::Amount,
+    ) -> anyhow::Result<PegOutFees> {
+        check_address(&address, self.cfg.network)?;
+
+        self.module_api
```

Comment:
Directly calling the module api here to me just seems like a shortcut to spawning a state machine which can access the api. 

I think its fine here since spawning a state machine is probably overkill for one API call. For future module developers though, do you think there's a worry here that APIs can be called without spawning a state machine? With this couldn't you implement logic that is very similar to the legacy client?

Reply:
This is very intentional: we need a way to get permission from the user to pay these fees, meaning if we'd fetch them inside the state machine we'd need to concoct a way to get user feedback in there.

Due to this having the fetching inside a state machine would be of little to no use since any sensible integrator will block on user feedback anyway and fetching fees does not give us any guarantees or requires us to take action later in regards to the federation, making all the possible benefits of doing so in a state machine moot.

Reply:
Yep I agree with all of that. My main concern was for new module developers who might not know when to use a state machine and when not to. With the module api directly exposed they now have the option of directly implementing their logic in the ClientModule. 

Agreed that this is the correct approach.